modules/lightning_features/lightning_media/modules/lightning_media_image/lightning_media_image.module

```diff
@@ -8,10 +8,50 @@
 use Drupal\Core\Entity\Display\EntityFormDisplayInterface;
 use Drupal\Core\Field\FieldStorageDefinitionInterface;
 use Drupal\Core\Form\FormStateInterface;
+use Drupal\image\Entity\ImageStyle;
+use Drupal\lightning_core\ConfigHelper as Config;
 use Drupal\lightning_core\OverrideHelper as Override;
 use Drupal\lightning_media_image\Plugin\MediaEntity\Type\Image;
 
 /**
+ * Implements hook_field_widget_info_alter().
+ */
+function lightning_media_image_field_widget_info_alter(array &$info) {
+  if (isset($info['image_widget_crop'])) {
+    // We cannot import the class with a use statement, because it will blow up
+    // if Image Widget Crop is not installed.
+    Override::pluginClass($info['image_widget_crop'], \Drupal\lightning_media_image\Plugin\Field\FieldWidget\ImageCropWidget::class);
```

Comment:
To get the behavior I wanted, I had no choice but to hijack the cropping widget plugin.